libmscore/property.cpp

```diff
@@ -332,6 +332,9 @@ static constexpr PropertyMetaData propertyList[] = {
       { Pid::TREMOLO_PLACEMENT,       false, "tremoloPlacement",      P_TYPE::INT,                 DUMMY_QT_TRANSLATE_NOOP("propertyName", "tremolo placement") },
       { Pid::HARMONY_TYPE,            true,  "harmonyType",           P_TYPE::INT,                 DUMMY_QT_TRANSLATE_NOOP("propertyName", "harmony type") },
 
+      { Pid::START_WITH_LONG_NAMES,   true,  "startWithLongNames",    P_TYPE::BOOL,                DUMMY_QT_TRANSLATE_NOOP("propertyName", "start with long names")  },
+      { Pid::START_WITH_MEASURE_ONE,  true,  "startWithMeasureOne",   P_TYPE::BOOL,                DUMMY_QT_TRANSLATE_NOOP("propertyName", "start with measure one") },
```

Comment:
`startWithMeasureOne` should certainly be linked beween score and part, to avoice confusion about what measure you'r at in a rehearsal. Not so sure though about `startWithLongNames`, this seems more of a layout thing? OTOH most parts have one instrument only and don't show instrtument names at all, so this may be pretty irrelevant.

Reply:
It looks like section break elements are not linked between master score and parts currently, so this will need to be fixed first to be linked between score and parts. Section breaks are linked though in between MM rest and the underlying measure, and in order for properties of section break in MM rest be actually applied in a score all these properties need to be linked.

Reply:
@dmitrio95 Were you suggesting `startWithLongNames` be not changed? Because I agree with @Jojo-Schmitz now, it is indeed a layout detail, maybe there are users who don't want long names in full score (for simplicity) but do want long names in parts. Just guessing, because I actually prefer to have consistency between score and parts.

Reply:
And yes, just checked, properties of section break aren't linked even though they're all `true`. Which is another bug.

Reply:
It probably is the section break itself that isn't linked, and in that case it doesn't matter at all whether its properties are

Reply:
Yes but once the bug gets fixed these properties do matter ;-)

Reply:
Indeed, but that bug's fix belongs here as well

Reply:
But, shouldn't there be a central process of handling whether these properties should be linked? (In other words, do you know where that process is?) The `true/false` fields are all defined in the same way, there shouldn't have separate functions dealing with these.

Reply:
And about the specific property: now there's two upvotes of `false` for `startWithLongNames`, what do you think, @dmitrio95?

Reply:
@Jojo-Schmitz Just checked, the properties _are_ linked, but not until the next layout.

Reply:
Ok, so yet another MM rest bug then

Reply:
And I'd not link `startWithLongNames`, as there is no way to have them different then, in case you want it.

Reply:
I don't see a problem with syncing an element, adding/removing one in score/part and one will be added/disappear in parts/master score, I don't know if that's what you want, or I misunderstood.

Reply:
Hmm, I guess then I missunderstood. What is the deal with section breaks and MM rests then?

Reply:
Create a score with only three measures, generate parts, put a section break at the end. Input note in the first measure (parts still have MM rest), change property of section break, it isn't linked. Input note in the second measure (parts no longer have MM rest), change property of section break, it is linked.